openedx/features/edly/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+
+
+class Organization(models.Model):
```

Comment:
I am not in favor of adding `edly` prefix in class name but I guess in order to avoid any confusion with edx organization we can name it `EdlyOrganization` and rename`SubOrganization` to `EdlySubOrganization`.

@ziafazal any suggestions?

Reply:
We are now going with `EdlyOrganization` and `EdlySubOrganization`.